applications/vessel-history/src/features/vessel-list-item/VesselListItem.tsx

```diff
@@ -70,18 +70,11 @@ const VesselListItem: React.FC<ListItemProps> = (props): React.ReactElement => {
               <label>{t('vessel.transmission_plural', 'transmissions')}</label>
               {vessel.firstTransmissionDate || vessel.lastTransmissionDate ? (
                 <Fragment>
-                  {t('common.from', 'from')}{' '}
-                  {vessel.firstTransmissionDate ? (
-                    <I18nDate date={vessel.firstTransmissionDate} />
-                  ) : (
-                    DEFAULT_EMPTY_VALUE
-                  )}{' '}
-                  {t('common.to', 'to')}{' '}
-                  {vessel.lastTransmissionDate ? (
-                    <I18nDate date={vessel.lastTransmissionDate} />
-                  ) : (
-                    DEFAULT_EMPTY_VALUE
-                  )}
+                  {t('vessel.transmissionRange', 'transmissionRange', {
+                    transmissions: vessel.posCount,
+                    start: vessel.firstTransmissionDate ? I18nDateAsString(vessel.firstTransmissionDate) :  DEFAULT_EMPTY_VALUE,
+                    end: vessel.lastTransmissionDate ? I18nDateAsString(vessel.lastTransmissionDate) :  DEFAULT_EMPTY_VALUE,
+                  })}
```

Comment:
Since I18nDateAsString is to be removed I'd replace it with `formatI18nDate`
```suggestion
                    start: vessel.firstTransmissionDate ? formatI18nDate(vessel.firstTransmissionDate) :  DEFAULT_EMPTY_VALUE,
                    end: vessel.lastTransmissionDate ? formatI18nDate(vessel.lastTransmissionDate) :  DEFAULT_EMPTY_VALUE,
                  })}
```